app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,14 @@
+class AccountActivationsController < ApplicationController
+    def edit
+        user = User.find_by(email: params[:email])
+        if user && !user.activated? && user.authenticated?(:activation, params[:id])
+            user.activate
+            #log_in user  <-------- todo
+            flash[:success] = "Account activated!"
```

Comment:
We're going to be using I18n everywhere so we can easily translate the interface into other languages. See [the Rails Internationalization (I18n) API Guide](https://guides.rubyonrails.org/i18n.html) for more information. In short, it means this line and others like it will look like this:

`flash[:success] = t('.account_activated')`